src/en/tut-lxd.md

```diff
@@ -0,0 +1,264 @@
+'lxd-test' controller listed.
+
+```no-highlight
+Use --refresh flag with this command to see the latest information.
+
+Controller  Model    User   Access     Cloud/Region         Models  Machines HA  Version
+lxd-test*   default  admin  superuser  localhost/localhost       2         1 none  2.0.0
+```
+
+A newly-created controller has two models: The 'controller' model, which should
+be used only for internal Juju management, and a 'default' model, which is
+ready for actual use. The controller model can only be destroyed by destroying
+the controller itself.
+
+The following command shows the currently active controller, model, and user:
+
+```bash
+juju whoami
+```
+
+In our example, the output should look like this:
+
+```no-highlight
+Controller:  lxd-test
+Model:       default
+User:        admin
+```
+
+## Deploy applications
+
+Juju is now ready to deploy applications from among the hundreds included in
+the [Juju charm store][charm store]. It is a good idea to test your new model.
+How about an MediaWiki site?
+
+```bash
+juju deploy cs:bundle/mediawiki-single
+```
+
+This will fetch a 'bundle' from the Juju store. A bundle is a pre-packaged set
+of applications, in this case 'MediaWiki', and a database to run it
+with. Juju will install both applications and leveraging the relation in the
+model between them, configures the wiki to use the newly created database
+instance. This is part of the magic of Juju: it isn't just about deploying
+software, Juju also knows how applications coordinate together.
+
+Installing shouldn't take long. You can check on how far Juju has got by running
+the command:
+
+```bash
+juju status
+```
+
+When the applications have been installed, the output of the above command will
+look something like this:
+
+![juju status](./media/juju-mediawiki-status.png)
+
+There is a lot of useful information there! The important parts for now are
+the 'App' section, which shows that MediaWiki and MySQL are installed, and the
+'Unit' section, which shows the IP addresses allocated to each. These addresses
+correspond to the subnet we created for LXD earlier on.
+
+Juju models which applications have firewalls that allow external access on
+clouds that support firewall controls. You can update the model by using the
+`juju expose <application>` command. In our case we would want to
+`juju expose mediawiki`.
+
+From the status output, we can see that the IP address for the MediaWiki
+site we have created is 10.154.173.2 Open a browser and enter that address
+to see the site.
+
+!["mediawiki site"](./media/juju-mediawiki-site.png)
+
+Congratulations, you have just deployed an application with Juju!
+
+!!! Note: To remove all the applications in the model you just created, it is
+often quickest to destroy the model with the command
+`juju destroy-model default` and then [create a new model][models].
+
+
+## Next Steps
+
+Now that you have a Juju-powered cloud, it is time to explore the amazing
```

Comment:
I think this should also be updated to reflect that this is no longer 'getting-started' . Perhaps something like

**Now that you have configured Juju to work with LXD images, you can test, tweak, develop and experiment with Juju charms rapidly, easily and without additional costs.** 

or similar